_locales/fr/messages.json

```diff
@@ -61,69 +61,69 @@
         "message": "Minutes"
     },
     "Secs": {
-        "message": "Seconds"
+        "message": "Secondes"
     },
     "Hide": {
-        "message": "Ne Montre Jaimais"
+        "message": "Toujours cacher"
```

Comment:
Is this understood as "Don't show again"?

Reply:
"Ne plus montrer" would be "Dont show again" and seems better indeed.

Reply:
Yep, I like "Ne plus montrer" a lot better.